content/applications/hr/payroll/configuration.rst

```diff
@@ -0,0 +1,359 @@
+all taxes, fees, and allowances for that particular country.  The localization section of the
+settings includes a detailed view of all benefits provided to employees.
+
+To access the settings, go to :menuselection:`Payroll --> Configuration --> Settings`. Whether or
+not payslips are posted in accounting, and whether SEPA payments are created, is selected here.
+
+.. image:: configuration/payroll-settings.png
+   :align: center
+   :alt: Settings available for payroll.
+
+Any country-specific localizations are set up in the  :guilabel:`Localization` section of the
+settings screen. All localization items are pre-populated when the country is specified during the
+creation of the database. It is not recommended to alter the localization settings unless
+specifically required.
+
+Work entries
+============
+
+A *work entry* is an individual record on an employee's timesheet. Work entries can be configured to
+account for all types of work and time off, such as `Attendance`, `Sick Time Off`, `Training`, or
+`Public Holiday`.
+
+Work entry types
+----------------
+
+When creating a work entry in the *Payroll* application, or when an employee enters information in
+the *Timesheets* application, a work entry type needs to be selected. The list of work entry types
+is automatically created based on localization settings set in the database.
+
+Each work entry type has a code to aid in the creation of payslips, and ensure all taxes and fees
+are correctly entered.
+
+Go to :menuselection:`Payroll --> Configuration --> Work Entry Types` to view the current work entry
+types available.
+
+.. image:: configuration/work-entry-types.png
+   :align: center
+   :alt: List of all work entry types currently available.
+
+New work entry type
+~~~~~~~~~~~~~~~~~~~
+
+To create a new :guilabel:`Work Entry Type`, click the :guilabel:`Create` button. Enter the
+information on the form:
+
+- :guilabel:`Work Entry Type Name`: The name should be short and descriptive, such as `Sick Time` or
+  `Public Holiday`.
+
+- :guilabel:`Code`: This code appears with the work entry type on timesheets and payslips. Since
+  the code is used in conjunction with the *Accounting* application, it is advised to check with
+  the accounting department for a code to use.
+
+- :guilabel:`Sequence`: The sequence determines the order that the work entry is computed in the
+  payslip list.
+
+- Check Boxes: If any of the items in the list applies to the work entry, check off the box by
+  clicking it. If :guilabel:`Time Off` is checked off, a :guilabel:`Time Off Type` field appears.
+  This field has a drop-down menu to select the specific type of time off, or a new type of time off
+  can be entered.
+
+.. image:: configuration/new-work-entry.png
+   :align: center
+   :alt: New work entry type box.
+
+- :guilabel:`Rounding`: The rounding method determines how timesheet entries are displayed on the
+payslip.
```

Comment:
Hi :)
Just passing by, as I saw there was a problem with a build:
![image](https://github.com/odoo/documentation/assets/46810843/e35b2b13-3c83-4c90-9abc-9a590aecca99)
As you can see when you build your doc locally, or by checking the ci build (the check made by robodoo on your pull request), there are two spaces missing at line 84
![image](https://github.com/odoo/documentation/assets/46810843/d886f082-930f-40fc-98c4-73fb5972c96f)
![image](https://github.com/odoo/documentation/assets/46810843/1b3dfd55-7b04-417e-9893-62f3635d8883)
```suggestion
- :guilabel:`Rounding`: The rounding method determines how timesheet entries are displayed on the
  payslip.
```

Have a good day!